airflow/cli/commands/task_command.py

```diff
@@ -170,8 +170,6 @@ def task_run(args, dag=None):
 
     task = dag.get_task(task_id=args.task_id)
     ti = TaskInstance(task, args.execution_date)
-    ti.refresh_from_db()
```

Comment:
All methods (_run_task_by_executor, _run_task_by_local_task_job, _run_raw_task) will call `refresh_from_db` before execution, so it seems there's no reason to do this here.